default/AI/AIDependencies.py

```diff
@@ -83,4 +83,47 @@
     "BLD_SHIPYARD_AST_REF",
 }
 
+FULL_REPAIR = 1e6  # arbitrary large number higher than any structure.
+FULL_FUEL = 1e6
+BASE_DETECTION = 25
+
+# Do specifiy hull detection only if it is irregular.
+# Organic growth has the structure (per_turn, maximum)
+HULL_EFFECTS = {
+    # Robotic line
+    "SH_ROBOTIC": {"REPAIR_PER_TURN": 2},
```

Comment:
are you referring to the key "REPAIR_PER_TURN"?  I guess I can see the minor benefit of not having to type the quotation marks, and more significantly, you then get spelling error checks from the IDE, which are not normally automatically done (at least not as strictly) for a string literal-- I just corrected a string-literal-key spelling error in the militaryAI code, so I do see a value to that.